git_machete/docs.py

```diff
@@ -5,6 +5,7 @@
     "add": "Add a branch to the tree of branch dependencies",
     "advance": "Fast-forward merge one of children to the current branch and then slide out this child",
     "anno": "Manage custom annotations",
+    "clean": 'Delete untracked and unmanaged branches also checkout user\' open PRs',
```

Comment:
```suggestion
    "clean": 'Delete untracked and unmanaged branches also check out user\'s open GitHub PRs',
```

Reply:
I've never seen `also` used to **link** two clauses in the sentence like that. It's rather bound to single clause... so e.g.

```
Delete untracked and unmanaged branches. Also, checkout user's open PRs
```

would be okay (albeit not very fitting the style of other one-line summaries here)

https://www.kaplaninternational.com/blog/learning-languages/eng/how-to-use-also-and-too-grammar-differences

Reply:
Hence, I suggest adding `and` to link the clauses first, and only then enhancing the latter clause with `also`:

````
<clause1> and also <clause2>
```

but not

````
<clause1> and <clause2>
```